source/heroes/ControllableHero.hx

```diff
@@ -0,0 +1,38 @@
+package heroes;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ControllableHero extends FlxSprite {
+    var DirectionX:Float;
+    var DirectionY:Float;
+
+    public function new(X:Float=0, Y:Float=0)
+    {
+        super(X, Y);
+        makeGraphic(25, 50, 0xFF006118);
+
+    }
+    override public function update(elapsed:Float)
+    {
+        super.update(elapsed);
```

Comment:
The ordering of your custom logic and this `super.update()` call can be pretty important. 

Remember what we discussed in class, that the `FlxSprite` object's `update` method is responsible for "drawing" the object on the screen. 

If we call this first, then we draw the character at one position, then the subsequent lines update their position.

Either ordering can work okay, but for most folks, it's more intuitive to do any custom update logic first, then call `super.update` at the end to draw the result of your changes.

Reply:
No action required, decision to change is up to you.